RuntimeUnityEditor.UMM/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+using RuntimeUnityEditor.Core;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("RuntimeUnityEditor.UMM")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Vanguard")]
+[assembly: AssemblyProduct("RuntimeUnityEditor.UMM")]
+[assembly: AssemblyCopyright("Copyright ©  2023")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("bd33ca70-24b9-4e80-908f-3e3a3c264904")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion(RuntimeUnityEditorCore.Version)]
```

Comment:
For consistency, also makes it easier to keep track of the version. If there's an UMM specific reason to not do this feel free to ignore.
```suggestion
[assembly: AssemblyVersion(RuntimeUnityEditorCore.Version)]
```

Reply:
This was for consistency ;) , it's copypasta from BepInEx AssemblyInfo.cs. I don't believe there is a requirement, UMM has something similiar to scriptEngine built in which used to rely on AssemblyVersion incrementing each time but that's no longer the case.

Do you want to go ahead with this change? Want to modify Bepin5's AssemblyInfo.cs to match?

Reply:
Oh, I didn't realize it was like this in the Bepin5 version, probably some leftover from old times. I'll change it in both after merging.